File: es_05.cpp
// 5. Inserisci 10 numeri,
// contare quanti numeri compresi tra 10 e 20 sono stati inseriti,
// contare anche i multipli di 3.

#include <iostream>
using namespace std;

int main() {
    int n, conta_10_20 = 0, conta_multipli_3 = 0;

    for (int i = 1; i <= 10; i++) {
        cout << "Inserisci un numero: ";
        cin >> n;

        if (n >= 10 && n <= 20) {
            conta_10_20++;
        }

        if (n % 3 == 0) {
            conta_multipli_3++;
        }
    }

    cout << "Numeri compresi tra 10 e 20: " << conta_10_20 << endl
         << "Multipli di 3: " << conta_multipli_3 << endl;

    return 0;
}

File: Python/es_05.py
from itertools import count

# 5. Inserisci 10 numeri,
# contare quanti numeri compresi tra 10 e 20 sono stati inseriti,
# contare anche i multipli di 3.

num_10_20 = mult_3 = 0

for i in count (start=1, step=1):
    n = int(
        input(
            'Inserisci un numero: '
        )
    )

    if 10 <= n <= 20:
        num_10_20 += 1
    
    if n%3 == 0:
        mult_3 += 1
    

    if i == 10: break

print(
    f'I numeri tra 10 e 20 sono {num_10_20}',
    f'I multipli di 3 sono {mult_3}', sep='\n'
)